README.md

```diff
@@ -268,16 +268,30 @@ $ mbed remove text-lcd
 
 After importing a program or creating a new one, you need to tell mbed CLI where to find the toolchains that you want to use for compiling your source tree.
 
-There are two ways to do this:
-* Through the mbed CLI configuration.
-* Via mbed_settings.py file in the root of your program, which is automatically created (if it doesn't already exist). 
+There are multiple ways to configure toolchain locations:
+* `mbed_settings.py` file in the root of your program. The tools will automatically create this file if it doesn't already exist. 
+* The mbed CLI configuration
+* Setting an environment variable
+* Adding directory of the compiler binary to your PATH
+
+Methods for configuring toolchains that appear earlier in the above list override methods that appear later.
+
+#### Through mbed_settings.py
+
+Edit `mbed_settings.py` to set your toolchain:
+
+* To use the [ARM Compiler toolchain](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler-5/downloads), set `ARM_PATH` to the *base* directory of your ARM Compiler installation (example: C:\Program Files\ARM\armcc5.06). The recommended version of the ARM Compiler toolchain is 5.06.
+* To use the [GCC ARM Embedded toolchain](https://launchpad.net/gcc-arm-embedded), set `GCC_ARM_PATH` to the *binary* directory of your GCC ARM installation (example: C:\Program Files\GNU Tools ARM Embedded\4.9 2015q2\bin). Use versions 4.9 of GCC ARM Embedded; version 5.0 or any version above might be incompatible with the tools.
+* To use the [IAR EWARM toolhain](https://www.iar.com/iar-embedded-workbench/#!?architecture=ARM), set `IAR_PATH` to the *base* directory of your IAR installation. Use versions 7.80 of IAR EWARM; prior versions might be incompatible with the tools.
+
+Since `mbed_settings.py` contains local settings (possibly relevant only to a single OS on a single machine), it should not be checked into version control. 
```

Comment:
Please change "Since" to "Because" because "since" implies time, and "because" implies causation.
Please change "it should not be checked" to you should not check it" for active voice.